patches.json

```diff
@@ -170,16 +170,16 @@
       "2.2.0 - 2.2.3": "MAGECLOUD-3611__multi_thread_scd__2.2.0.patch",
       "2.2.4 - 2.2.9": "MAGECLOUD-3611__multi_thread_scd__2.2.4.patch",
       "2.3.0 - 2.3.1": "MAGECLOUD-3611__multi_thread_scd__2.3.0.patch",
-      "2.3.2": "MAGECLOUD-3611__multi_thread_scd__2.3.2.patch"
+      ">=2.3.2 <2.3.3": "MAGECLOUD-3611__multi_thread_scd__2.3.2.patch"
     },
     "setup:upgrade returns error code if app:config:import failed": {
       "2.2.0": "MAGECLOUD-3806__error_code_fix_for_setup_upgrade__2.2.0.patch",
       "2.2.1 - 2.2.9": "MAGECLOUD-3806__error_code_fix_for_setup_upgrade__2.2.1.patch",
-      "2.3.0 - 2.3.2": "MAGECLOUD-3806__error_code_fix_for_setup_upgrade__2.3.0.patch"
+      ">=2.3.0 < 2.3.3": "MAGECLOUD-3806__error_code_fix_for_setup_upgrade__2.3.0.patch"
```

Comment:
Redundant space before 2.3.3 (not sure if composer handles it)

Reply:
removed